Assignment2/ziader-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,26 @@
+{
+	"@context": {
+		":":"http://example.org/#"
+		"foaf":"http://xmlns.com/foaf/0.1/"
+		"rdfs":"http://www.w3.org/2000/01/rdf-schema#"
+	},
+   "@id":":Class01",
+   "@type":"foaf:Class",
+   "rdfs:includes":{
+	  "@id":":Computer101",
+	  "@type":"foaf:Class",
+      "rdfs:hasOwner":{
+		"@id":":User10A",
+	    "@type":"foaf:Class",
+		"rdfs:hasName":"Pedro"
+	  }
+	  "@id":":Sensor029",
+	  "@type":"foaf:Class",
+      "rdfs:hasMeasurement":{
+		"@id":":Measurement8401",
+	    "@type":"foaf:Class",
+		"rdfs:hasTemperature":"29",
+		"rdfs:atTime":"2010-06-12T12:00:12"
+	  }
+   }
+}
```

Comment:
As you have two resources that are related to the "rdfs:includes" property, you need to define them as an array thus they must be defined between [ ]. Please add it ->  "rdfs:includes":[{ and the ] where corresponds. 

In addition, you are using the same concept to define the type of resources. Please add a proper type according to each case e.g. "@id":":Computer101" should be "@type":"foaf:Computer". Also, be careful when reusing vocabularies because in FOAF there are not exist such concepts.